exercises/dnd-character/test/dnd_character_test.exs

```diff
@@ -0,0 +1,110 @@
+defmodule DndCharacterTest do
+  use ExUnit.Case
+
+  import DndCharacter, only: [modifier: 1, ability: 0, character: 0]
+
+  # canonical data version: 1.1.0
+
+  describe "ability modifier" do
+    # @tag :pending
+    test "for score 3 is -4" do
+      assert modifier(3) == -4
```

Comment:
I think to test for type, it should be `===` vs `==`

Reply:
Sure, strict equality seems like a good fit here.